lua/main_config.lua

```diff
@@ -5,6 +5,8 @@
 package.loaded["gatekeeper/staticlib"] = nil
 require "gatekeeper/staticlib"
 
+SUCCESS = 1
```

Comment:
Instead of defining a global variable, just return `1` and add a comment explaining that it's a required value to initialize Gatekeeper.